### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by rouf7** [phoomin2012/travian-kingdom#20]
hi i am  the guy from skype that we talked before some months .. are you still working in the project? because i am messaging you and you dont answer me

**Comment by mosessaleh** [comment]
Hi
I am sorry because i did not answer.
I changed my computer and did not logged in all websites that i logged
befor.
I will check it today.

And yes i am still want to work on the project.

Have a nice day.
Mousa

On Dec 1, 2017 02:49, "rouf7" <notifications@github.com> wrote:

hi i am the guy from skype that we talked before some months .. are you
still working in the project? because i am messaging you and you dont
answer me

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/phoomin2012/travian-kingdom/issues/20>, or mute the
thread
